README.md

```diff
@@ -156,25 +156,6 @@ zypper in eza
 
 The preceding repository also contains the Bash, Fish, and Zsh completions.
 
-### Fedora
```

Comment:
Wouldn't it be enough to change this to openSUSE? Might potentially also add https://bugzilla.redhat.com/show_bug.cgi?id=2238264 under fedora.